automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala

```diff
@@ -220,9 +216,9 @@ class LaunchAnalysisModal(implicit webDriver: WebDriver) extends FireCloudView {
       click on (await enabled launchAnalysisButtonQuery)
     }
 
-    def isNoDefaultEntitiesMessagePresent(): Boolean = {
-      await enabled emptyDefaultEntitiesMessageQuery
-      find(emptyDefaultEntitiesMessageQuery).size == 1
+    def getSelectedEntityText(): String = {
+      await enabled selectedEntityQuery
+      find(selectedEntityQuery).fold("")(_.text)
```

Comment:
I go back and forth between using `fold` and `map ... getOrElse` for `Option`. I should probably just get used to `fold`.